frontend/src/Components/Blogs/BlogCard.tsx

```
@@ -0,0 +1,105 @@
+
+const BlogCard = ({ blog }: { blog: Blog }) => {
+	const router = useRouter()
+	const { data: user } = useAccountStore()
+	const likeBlog = useLikeBlog()
+	return (
+		<article
+			key={blog.id}
+			className='bg-white rounded-xl shadow-lg overflow-hidden hover:shadow-xl duration-300'
+		>
+			<div className='h-48 bg-gradient-to-br from-main to-secondary flex items-center justify-center'>
+				{blog.imageUrls && blog.imageUrls[0] ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)